prisma/seed/voyage.ts

```diff
@@ -88,4 +88,34 @@ export const populateVoyages = async () => {
             showcasePublishDate: new Date("2024-10-28T06:59:59.000Z"),
         },
     });
+    await prisma.voyage.create({
+        data: {
+            number: "52",
+            status: {
+                connect: {
+                    name: "Active",
+                },
+            },
+            startDate: new Date("2024-11-04T17:00:00.000Z"),
+            endDate: new Date("2024-12-15T04:59:59.000Z"),
+            soloProjectDeadline: new Date("2024-10-25T06:59:59.000Z"),
+            certificateIssueDate: new Date("2024-12-21T06:59:59.000Z"),
+            showcasePublishDate: new Date("2024-12-27T06:59:59.000Z"),
+        },
+    });
+    await prisma.voyage.create({
+        data: {
+            number: "53",
+            status: {
+                connect: {
+                    name: "Upcoming",
+                },
+            },
+            startDate: new Date("2025-01-06T17:00:00.000Z"),
+            endDate: new Date("2025-02-16T04:59:59.000Z"),
```

Comment:
please update endDate, soloprojectDeadline, CertificationIssueDate and showcasePublishDat as above (v52) - you can see here for more detail (https://github.com/chingu-x/chingu-dashboard-be/wiki/Timezone)